### PR TITLE
Update readme instructions & version

### PR DESCRIPTION
**Pull request by tib**
Hello,

I'd like to contribute a small readme update with the following changes:

- package version is bumped to 2.0.0, since the current version is at 2.3.0
- target dependency snippet is now a one-liner, so users can simply copy & paste it
- added an extra code section with a sample `Package.swift` file, using ServiceLifecycle

Best,
Tib

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,13 +17,35 @@ If you have a server-side Swift application or a cross-platform (e.g. Linux, mac
 To add a dependency on the package, declare it in your `Package.swift`:
 
 ```swift
-.package(url: "https://github.com/swift-server/swift-service-lifecycle.git", from: "2.0.0-alpha.1"),
+.package(url: "https://github.com/swift-server/swift-service-lifecycle.git", from: "2.0.0"),
```
Comment (FranzBusch) [inline README.md:20]:
```suggestion
.package(url: "https://github.com/swift-server/swift-service-lifecycle.git", from: "2.3.0"),
```

File: README.md [hunk 2]
```diff
@@ -17,13 +17,35 @@ If you have a server-side Swift application or a cross-platform (e.g. Linux, mac
...
+Example `Package.swift` file with `ServiceLifecycle` as a dependency:
+
+```swift
+// swift-tools-version:5.9
+import PackageDescription
+
+let package = Package(
+    name: "my-application",
+    dependencies: [
+        .package(url: "https://github.com/swift-server/swift-service-lifecycle.git", from: "2.0.0"),
```
Comment (FranzBusch) [inline README.md:38]:
```suggestion
        .package(url: "https://github.com/swift-server/swift-service-lifecycle.git", from: "2.3.0"),
```

**Review by FranzBusch — COMMENTED** [review]
Thanks for updating this. Do you mind adding this to the index document of our Docc article as well?

**Comment by tib** [comment]
> Thanks for updating this. Do you mind adding this to the index document of our Docc article as well?

I just did that too, using v2.3.0, I hope this is what you were thinking about.

**Review by FranzBusch — APPROVED** [review]
Great. Thanks for the work!

**Comment by FranzBusch** [comment]
@swift-server-bot test this please

**Comment by FranzBusch** [comment]
@swift-server-bot test this please

**Review by FranzBusch — APPROVED** [review]

**Comment by FranzBusch** [comment]
@swift-server-bot test this please

**Pull request merged**